### ISSUE TITLE
Broken Upgrade from 5103 to 5108

### ISSUE THREAD

**Issue by thet** [plone/Products.CMFPlone#2143]
looks like its the upgrade step: "Move safe_html settings from portal_transforms to Plone registry"

```
Starting the migration from version: 5103
Role / permission map imported.
Toolset imported.
Ran upgrade step: Run to51beta1 upgrade profile.
Removed object/displayContentsTab from action at /iaem/iaemportal/portal_actions/object/folderContents
Ran upgrade step: Remove displayContentsTab from action expressions
Ran upgrade step: Miscellaneous
Role / permission map imported.
Ran upgrade step: Run to51beta3 upgrade profile.
Role / permission map imported.
Removed record plone.resources/tinymce-layer.init.
Removed record plone.resources/tinymce-layer.export.
Removed record plone.resources/tinymce-layer.deps.
Removed record plone.resources/tinymce-layer.conf.
Removed record plone.resources/tinymce-layer.url.
Removed record plone.resources/tinymce-layer.css.
Removed record plone.resources/tinymce-layer.js.
Catalog imported.
Field types updated on social media schema
Ran upgrade step: Update social media fields
Role / permission map imported.
Removed record plone.resources/tinymce-layer.init.
Removed record plone.resources/tinymce-layer.export.
Removed record plone.resources/tinymce-layer.deps.
Removed record plone.resources/tinymce-layer.conf.
Removed record plone.resources/tinymce-layer.url.
Removed record plone.resources/tinymce-layer.css.
Removed record plone.resources/tinymce-layer.js.
Catalog imported.
Ran upgrade step: Run to51beta4 upgrade profile.
Ran upgrade step: Fix double shared maxage
Reindexed `mime_type` for 18829 items
Ran upgrade step: Reindex mime_type
Role / permission map imported.
Actions tool imported.
Ran upgrade step: Run to51beta5 upgrade profile.
Upgrade aborted. Error:
Traceback (most recent call last):
  File "/home/iaem/iaem-buildout/eggs/Products.CMFPlone-5.1rc1-py2.7.egg/Products/CMFPlone/MigrationTool.py", line 268, in upgrade
    step['step'].doStep(setup)
  File "/home/iaem/iaem-buildout/eggs/Products.GenericSetup-1.8.8-py2.7.egg/Products/GenericSetup/upgrade.py", line 166, in doStep
    self.handler(tool)
  File "/home/iaem/iaem-buildout/eggs/plone.app.upgrade-2.0.7-py2.7.egg/plone/app/upgrade/v51/betas.py", line 188, in move_safe_html_settings_to_registry
    settings = registry.forInterface(IFilterSchema, prefix='plone')
  File "/home/iaem/iaem-buildout/eggs/plone.registry-1.1.2-py2.7.egg/plone/registry/registry.py", line 82, in forInterface
    name
KeyError: 'Interface `Products.CMFPlone.interfaces.controlpanel.IFilterSchema` defines a field `valid_tags`, for which there is no record.'
End of upgrade path, main migration has finished.
The upgrade path did NOT reach current version.
Migration has failed
```

**Comment by pbauer** [comment]
Already mentioned in https://community.plone.org/t/plone-5-1rc1-soft-released/4850/3 and fixed in https://github.com/plone/plone.app.upgrade/pull/135

**Comment by hvelarde** [comment]
it's not fixed as I described in #2129.
